Source/ProgressionSystemRuntime/Private/Widgets/PSOverlayWidget.cpp

```diff
@@ -4,13 +4,119 @@
 #include "Widgets/PSOverlayWidget.h"
 
 #include "Components/Image.h"
+#include "Curves/CurveFloat.h"
+#include "Data/PSDataAsset.h"
 
 #include UE_INLINE_GENERATED_CPP_BY_NAME(PSOverlayWidget)
 
-// Sets the visibility of the background overlay and lock icon.
-void UPSOverlayWidget::SetOverlayVisibility(ESlateVisibility VisibilitySlate)
+// Sets the visibility of the overlay elements and playing fade animation if needed
+void UPSOverlayWidget::SetOverlayVisibility(ESlateVisibility VisibilitySlate, bool bShouldPlayFadeAnimation/* = false*/)
```

Comment:
`bool bShouldPlayFadeAnimation/* = false*/` says it's `false` by default, but in .h it does not have any default value set

Reply:
should be there: 
![image](https://github.com/user-attachments/assets/0a5ff8ee-7868-4163-bbff-f7899c0d466a)

Reply:
I'm talking about SetOverlayVisibility function argument. In cpp definition you write comment /* = false */ while in .h declaration this function argument does not have default value. Or make your argument to be `= false`, or remove comment from .cpp declaration

Reply:
added: https://github.com/h4rdmol/ProgressionSystem/pull/20/commits/7d46e647ccff96854bf0a2fee08077c51d7985b7